public/css/user.css

```diff
@@ -3,4 +3,8 @@ hr {
     background-color: gray;
     width: 60%;
     border: none;
+}
+
+.m-0 {
```

Comment:
It's still here. `m-0` is a class in bootstrap that you can just call to have a style with margin of all sides to 0, you do not need to set in css file anymore.